Example/Marker-Example.xcodeproj/project.pbxproj

```diff
@@ -446,12 +414,64 @@
 		271C85CF1ED4E3F000F8BBBB /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING = YES;
```

Comment:
What are all these additions for?

Reply:
Upgrading the project to support Swift 4.2 may have included the setting.
Could take it out if not required